ci-operator/step-registry/telcov10n/functional/cnf-ran/eco-gotests-compact/telcov10n-functional-cnf-ran-eco-gotests-compact-commands.sh

```
@@ -2,129 +2,136 @@
-    echo "Processing complete. Check \"${dest_file}\""
+  find "$directory" -type f | while IFS= read -r filename; do
+    if [[ $filename == *"secretsync-vault-source-path"* ]]; then
+      continue
+    fi
+    local content
+    content=$(cat "$filename")
+    local varname
+    varname=$(basename "${filename}")
+    if [[ "$content" == *$'\n'* ]]; then
+      echo "${varname}: |"
+      echo "$content" | sed 's/^/  /'
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)